### PR TITLE
Move taskcluster CI decision task to generic-worker

### PR DESCRIPTION
**Pull request by petemoore**
Seeing if I can switch our own docker-worker CI decision task to generic-worker, using d2g tool...

Let's see how many attempts this takes, at the same time I've tried to reformat the output a little bit to make it easier to read, and due to all the encodings inside encodings, this may take a few attempts to get right...

### CODE REVIEW

File: .taskcluster.yml [hunk 1]
```diff
@@ -167,41 +168,62 @@ tasks:
...
+                              -e "TASKCLUSTER_HEAD_REV=$${TASKCLUSTER_HEAD_REV}" \
+                              -e "TASKCLUSTER_HEAD_TAG=$${TASKCLUSTER_HEAD_TAG}" \
+                              -e "TASKCLUSTER_PIP_REQUIREMENTS=$${TASKCLUSTER_PIP_REQUIREMENTS}" \
+                              -e "TASKCLUSTER_PROXY_URL=$${TASKCLUSTER_PROXY_URL}" \
+                              -e "TASKCLUSTER_PULL_REQUEST_NUMBER=$${TASKCLUSTER_PULL_REQUEST_NUMBER}" \
+                              -e "TASKCLUSTER_REPOSITORY_TYPE=$${TASKCLUSTER_REPOSITORY_TYPE}" \
+                              -e "TASKCLUSTER_ROOT_URL=$${TASKCLUSTER_ROOT_URL}" \
+                              -e "TASKCLUSTER_WORKER_LOCATION=$${TASKCLUSTER_WORKER_LOCATION}" \
+                              -e "TASK_ID=$${TASK_ID}" \
+                              "mozillareleases/taskgraph:decision-d1ddb5593679cec9d7d2cdd8e9b8e2228b14ebe9b844411276cc3cfe37d189aa@sha256:b236321d98367f9cdbbd28c47bad7c0df34884897b75d266185d2e9e569ac0fa" \
```
Comment (matt-boris) [inline .taskcluster.yml:196]: Now's probably a good time to bump to [latest decision image](https://hub.docker.com/layers/mozillareleases/taskgraph/decision-8af16934fd68303cebf9735705515cb48df542d251c04e921c7ac50564faac32/images/sha256-2965f42c2e91fc66ae7bd27d5dff88297225a90d04ebc6de23c538e3933d4049?context=explore)

```suggestion
                              "mozillareleases/taskgraph:decision-8af16934fd68303cebf9735705515cb48df542d251c04e921c7ac50564faac32@sha256:2965f42c2e91fc66ae7bd27d5dff88297225a90d04ebc6de23c538e3933d4049" \
```

File: changelog/TotHjZ6OTS6wbCFSoKtoNA.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+audience: general
+level: silent
```
Comment (matt-boris) [inline changelog/TotHjZ6OTS6wbCFSoKtoNA.md:2]: What do you think about a patch for developers?

**Review by matt-boris — COMMENTED** [review]
_So cool!!!_ 🎉

Left a couple comments/suggestions :)

File: changelog/TotHjZ6OTS6wbCFSoKtoNA.md (hunk 2 above)
Comment (petemoore) [reply changelog/TotHjZ6OTS6wbCFSoKtoNA.md:2]: Update: we discussed over slack, and decided that the changelog should only affect the release notes, and this PR won't affect the release, so it probably is ok as silent.

File: .taskcluster.yml (hunk 1 above)
Comment (matt-boris) [reply .taskcluster.yml:196]: Created https://github.com/taskcluster/taskcluster/pull/6068

**Pull request merged**